study.md

```diff
@@ -57,7 +57,11 @@ Which ActiveRecord method creates new objects? Does this method persist objects
 as rows in the database, or is there another method required for persistence?
 
 ```md
-<!-- your response here -->
+<!--
+class Person < ApplicationRecord
+end
+Using this, you will have the ability to map the columns of each row in that table with the attributes of the instances of the Person model.
+ -->
 ```
```

Comment:
you can use .create or .new to make a new object
create makes and saves where as .new only creates